_posts/2020-04-20-ImageRecognition.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "Teaching Computers to Understand Pictures"
+author: Arindam
+categories: [ Image Recognition, AI, ML, DL, Neural Networks, CNN ]
+image: assets/images/imgrecog.png
+wide_image: assets/images/imgrecog.png
+thumb_image: assets/images/imgrecog.png
+last_added: false
+hidden: false
+---
+Let�s look at two images to get started with:  
+
+![](/assets/images/dog.jpg)  
+
+![](/assets/images/tree.jpg)  
+
+Ask yourself, what do these images show?  
+It�s not di?cult to see that the ?rst image is that of a �dog� and the other one is that of a �tree�!  
```

Comment:
I don't understand di?cult and ?rst. Please rectify these and probably others from the document.